internal/controller/controllers/common.go

```diff
@@ -16,6 +16,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const mirrorRegistryRefCertKey = "ca-bundle.crt"
+const mirrorRegistryRefRegistryConfKey = "registries.conf"
+const mirrorRegistryConfVolume = "mirror-registry-conf"
+const mirrorRegistryCertVolume = "mirror-registry-ca"
```

Comment:
https://golang.org/doc/effective_go#constants and https://golang.org/doc/effective_go#commentary make me think that we should be doing something like:

```suggestion
// important keys for handling a mirror registry
const (
	mirrorRegistryRefCertKey = "ca-bundle.crt"
	mirrorRegistryRefRegistryConfKey = "registries.conf"
	mirrorRegistryConfVolume = "mirror-registry-conf"
	mirrorRegistryCertVolume = "mirror-registry-ca"
)
```